presto-orc/src/main/java/com/facebook/presto/orc/reader/LongSelectiveStreamReader.java

```diff
@@ -49,30 +53,30 @@ public LongSelectiveStreamReader(
             OrcAggregatedMemoryContext systemMemoryContext,
             boolean isLowMemory)
     {
-        this.streamDescriptor = requireNonNull(streamDescriptor, "streamDescriptor is null");
-        directReader = new LongDirectSelectiveStreamReader(streamDescriptor, filter, outputType, systemMemoryContext.newOrcLocalMemoryContext(LongSelectiveStreamReader.class.getSimpleName()));
```

Comment:
OrcLocalMemoryContext name will change, but this should not be a big deal

Reply:
SliceReader uses the correct class names, but I believe long reader has a bug where it is used incorrectly, so fixed it.